### ISSUE TITLE
PhantomJS archiving

### ISSUE THREAD

**Issue by kpyorala** [marcbachmann/node-html-pdf#404]
https://github.com/ariya/phantomjs/issues/15344

Not the best news for this project. Maybe this could be used as replacement:

https://github.com/laurentj/slimerjs 

"SlimerJS provides the same API of PhantomJS"

**Comment by marcbachmann** [comment]
Yeah, I've suspected that since a while. Some contributor stepped back a while ago and since then it was only in "maintenance" mode.

I'd rather migrate it to chromium.
The header and footer templates just recently got introduced in puppeteer but in addition to that we'll need some additional logic (which generates css) to support custom headers on pages with a specific index.

**Comment by rbecheras** [comment]
@marcbachmann I'd rather thought like @kpyorala : migrate to slimmer because it could be almost a dependency migration, knowing slimmer and phantom shares the same API. Plus, it could be a fix for some phantomjs specific issues.

And in a second time, adding puppeteer as an option let to the user could be an awesome thing.

What about you ?

**Comment by kpyorala** [comment]
For me both options are ok, important part is that this library continues to work, as we have just implemented fleet management system reporting with this :) what ever option you choose I can help at least as a tester and maybe some development too if needed.

**Comment by shirbr510** [comment]
I see that his discussion has no new response since march...
any decision on where will the wind blow?
